app/code/Magento/ImportService/Model/ImportRestartOperation.php

```
@@ -0,0 +1,35 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\ImportService\Model;
+
+use Magento\ImportService\Api\ImportRestartOperationInterface;
+
+/**
+ * Class ImportRestartOperation
+ *
+ * @package Magento\ImportService\Model
+ */
+class ImportRestartOperation implements ImportRestartOperationInterface
+{
+    /**
+     * restart failed operation.
+     *
+     * @param int $uuid
+     * @param string $serializedData
+     * @return bool
+     */
+    public function execute(int $uuid, string $serializedData)
+    {
+        try
+        {
+            return true;
+        } catch (\Exception $e) { }
+
+        return false;
```

Can you please change response from true/false to empty array
[]
Also lets throw an exception. Not a good to catch en exception without any action.

---

I have changed return type from bool to array i.e. []. Also, as there is no operation performed at this moment, I have removed an additional try/catch statement to avoid any confusion.